### DISCUSSION TITLE
About Page

### DISCUSSION THREAD

**Discussion by Preinspiron** [AlexSBolotov/react-bc50-1-budget-boss#124] [Ideas]
Teamlead Frontend @ElenaKorenieva 

[Макет Figma](https://www.figma.com/file/Qm2iIU9GZ6CR8bKqwy9DgR/Budget-Boss-(Copy)?type=design&node-id=601-2&t=jLAniQmXSpfC0ZXc-0)

**Comment by Preinspiron** [comment]
привет еще раз. Витя  сообщил что хочет во фронт. И нам для 'about' может какой то макет набросать в figme ?

**Comment by ElenaKorenieva** [comment]
Давай, мне подключиться на митинг?
Ты в телеграмм не отвечаешь))

**Reply by Preinspiron** [reply]
давай
